client/src/components/map/Map.jsx

```diff
@@ -44,15 +44,28 @@ function Map({ stations, fuelValue }) {
                   closeButton={false}
                   closeOnClick={false}
                   popupOpen={true}>
-                  <PriceWrapper>
+                  <p>
                     <Price price={station.fuelPrices[fuelValue].price} />
-                  </PriceWrapper>
+                  </p>
                   <Link to={`/${station.id}`}>Mehr Anzeigen</Link>
                 </NewPopup>
               </MyMarker>
             );
           })}
       </MarkerCluster>
+
+      {latitude && longitude !== null && (
+        <MyMarker position={[latitude, longitude]}>
+          <NewPopup
+            autoClose={false}
+            closeOnEscapeKey={false}
+            closeButton={false}
+            closeOnClick={false}
```

Comment:
Is it possible that these props have a default value "false"? if so, you don't need to add them here with the value "false".

Reply:
Default is true, so we had to set it to false. But thanks for your hint. 😇